src/darker/command_line.py

```diff
@@ -49,4 +49,20 @@ def parse_command_line(argv: List[str]) -> Namespace:
     parser.add_argument(
         "--version", action="store_true", help="Show the version of `darker`"
     )
+    parser.add_argument(
+        "-S",
+        "--skip-string-normalization",
+        action="store_true",
+        dest="skip_string_normalization",
+        default=False,
+        help="Sets the `--skip-string-normalization` flag for `black`",
```

Comment:
For other options, I've used the imperative mood ("sort", "show", "reduce" instead of "sorts", "shows", "reduces").
```suggestion
        help="Set the `--skip-string-normalization` flag for `black`",
```
Or, why not just copy the description from black?
```suggestion
        help="Don't normalize string quotes or prefixes",
```